newIDE/app/src/Profile/CreateProfile.js

```diff
@@ -12,11 +12,17 @@ type Props = {
   message?: React.Node,
   onLogin: () => void,
   onCreateAccount: () => void,
+  justifyContent?: 'center',
```

Comment:
It feels weird to have just one possibility. It's the same as having true/false. I don't know if it's a good practice to have it like that.
But i'm not clear on what would be a good solution given Line prop `justifyContent: string`

Reply:
I've used this in another component in the sense that it's telling you "either don't specify it (probably by default)" or you can specify it but it's so exceptional that I give you just one possibility (center the text).
In the sense: please don't think of abusing this to use "flex-end", that's a no! :D